int8 LOGGER_COMMAND_RUN=1
int8 LOGGER_COMMAND_STOP=2
int8 LOGGER_COMMAND_SAVE=3
int8 logger_command